src/main/java/org/broadinstitute/consent/http/db/SamDAO.java

```diff
@@ -97,17 +97,13 @@ public UserStatus postRegistrationInfo(AuthUser authUser) throws Exception {
     HttpResponse response = executeRequest(request);
     String body = response.parseAsString();
     if (!response.isSuccessStatusCode()) {
-      if (HttpStatusCodes.STATUS_CODE_CONFLICT == response.getStatusCode()) {
-        throw new ConsentConflictException("User exists in Sam: " + authUser.getEmail());
-      } else {
-        String errorMsg = String.format("Error posting user registration information to Sam. Email [%s]. Status Code [%s]; Status Message [%s];  ",
+      String message = JsonParser.parseString(body).getAsJsonObject().get("message").getAsString();
```

Comment:
This looks a brittle to me - we're expecting that ALL non-successful cases will always have a `message` object. I think it will be safer to pass through the whole body instead of trying to parse it.

Reply:
okay true, I think it would be preferable to have just the message so maybe I could try to get the message and if it doesn't exist I could return the whole body